numerics/poisson_series_test.cpp

```diff
@@ -199,11 +205,63 @@ TEST_F(PoissonSeriesTest, PointwiseInnerProduct) {
               AlmostEquals(5 * Metre * Metre, 0));
 }
 
+TEST_F(PoissonSeriesTest, Fourier) {
+  Degree0::Polynomial constant({1.0}, t0_);
+  AngularFrequency const ω = 4 * Radian / Second;
+  PoissonSeries<Displacement<World>, 0, HornerEvaluator> signal(
+      constant * Displacement<World>{},
+      {{ω,
+        {/*sin=*/constant *
+             Displacement<World>({2 * Metre, -3 * Metre, 5 * Metre}),
+         /*cos=*/constant *
+             Displacement<World>({-7 * Metre, 11 * Metre, -13 * Metre})}}});
+  // Slice our signal into segments short enough that one-point Gauss-Legendre
+  // (also known as midpoint) does the job.
+  constexpr int segments = 100;
+  PiecewisePoissonSeries<Displacement<World>, 0, HornerEvaluator> f(
+      {t0_, t0_ + π * Second / segments}, signal);
+  for (int k = 1; k < segments; ++k) {
+    f.Append({t0_ + k * π * Second / segments,
+              t0_ + (k + 1) * π * Second / segments},
+             signal);
```

Comment:
I would call `AtOrigin` to put the origin of `signal` in the middle of the interval, since it would correspond more closely to what we do in real life.